### ISSUE TITLE
v4.8 Nefarious Problems with tinymce.triggerSave()

### ISSUE THREAD

**Issue by donShakespeare** [tinymce/tinymce#4498]
**Issue Demo:** http://fiddle.tinymce.com/owgaab/2

My scenario:
I have about 40+ inline editors, on one page.
Before I **submit** the form, I take all the content of the editors and place them in one same place.
Works beautifully!

Enter v4.8 ...

I **submit** my parent form now, all earth freezes over.
1., `tinymce.triggerSave()` fires (without my permission, as usual),
2.  and when it does fire it goes back to each editor to eat up/distort/cleanup my content :(

To reproduce, use the entire code below in a fiddle
**Issue Demo:** http://fiddle.tinymce.com/owgaab/2

```html
<script type="text/javascript">

//SOLUTION #1 (destroy save for all)
  //tinymce.triggerSaveBK = tinymce.triggerSave;
  //tinymce.triggerSave = function(){alert("Aloha! triggerSave intercepted :) ... we are good now");};

tinymce.init({
  selector: "div",
  inline: true, //issue is with inline 
  plugins: "noneditable code contextmenu",
  contextmenu: "code",
  menubar: false,
  toolbar: false,
  valid_elements: "*[*]", //kills off all empty paragraphs and stuff
  noneditable_noneditable_class: "do-not-edit",
  setup: function(editor){
    editor.on("SaveContent", function(e){
      alert("SaveContent Event was reached, check the contenteditable=false paragraph");
    });
//SOLUTION #2 (destroy save for each)
    /*editor.on("init", function(){
      editor.saveBK = editor.save;
      editor.save = function(){
        console.log("Aloha! editor.save() intercepted :) ... we are good now");
      };
    });*/
  }
});
</script>

<form method="post" action="dump.php">
  <input type=submit value=SUBMIT onclick='tinymce.triggerSave(); return false;' />
  <div style="min-height:100px;border:2px blue dotted;">
    <p><b>Broken from version 4.8 (works in cloud-stable)</b><br><br>Press <b>SUBMIT:</b><br>
    1. empty paragraphs get removed -thanks to <b>valid_elements: "*[*]"</b><br>
    2. you lose all contenteditable=false attributes below<br>
    3. because from 4.8 form <b>tinymce.triggerSave();</b> seems to be doing a lot more than it should<br>
    4. <b>SOLUTION:</b> empty the <b>tinymce.triggerSave()</b> function</p>
    <p></p><p></p><p></p><p></p> <p></p>
    <p class="do-not-edit">Click this to test contenteditable="false" Paragraph</p>
  </div>
  
</form>
```

**Comment by donShakespeare** [comment]
I find the second solution more effective and less invasive

```
//SOLUTION #2 (destroy save for each)
    editor.on("init", function(){
      editor.saveBK = editor.save;
      editor.save = function(){
        console.log("Aloha! editor.save() intercepted :) ... we are good now");
      };
    });
```

**Comment by fyrkant** [comment]
This has been fixed in newly released 4.8.1 version.

**Comment by donShakespeare** [comment]
Awesome work!
